src/infra/vhost/virtualHostQueryRepo.go

```diff
@@ -492,3 +527,33 @@ func (repo VirtualHostQueryRepo) GetMappingById(
 
 	return mapping, errors.New("MappingNotFound")
 }
+
+func (repo VirtualHostQueryRepo) CheckDomainOwnership(
```

Comment:
Se o retorno é boleano, a função deve geralmente começar com Is ou Are. IsDomainPontingToServer por exemplo seria um nome mais direto com base na intenção, mas o chat talvez te dê um nome melhor.

Reply:
Troquei para "IsDomainMappedToServer", assim atribui a ideia dos mappings que nós temos na aplicação e que utilizamos na criação do processo de validação de ownership.